backend/src/event/EVENT_ADMIN_CONTRIBUTION_LINK_DELETE.ts

```diff
@@ -0,0 +1,20 @@
+import { User as DbUser } from '@entity/User'
+import { ContributionLink as DbContributionLink } from '@entity/ContributionLink'
+import { Event as DbEvent } from '@entity/Event'
+import { Event, EventType } from './Event'
+
+export const EVENT_ADMIN_CONTRIBUTION_LINK_DELETE = async (
+  moderator: DbUser,
+  contributionLink: DbContributionLink,
+): Promise<DbEvent> =>
+  Event(
+    EventType.ADMIN_CONTRIBUTION_LINK_DELETE,
+    { id: 0 } as DbUser,
+    moderator,
+    null,
+    null,
+    null,
+    null,
+    null,
+    contributionLink,
```

Comment:
is the amount for this event-type not important?

Reply:
I believe the important info is that the link was deleted - the amount seems not very relevant in this circumstance